samples/test/sample.test.js

```diff
@@ -71,4 +79,22 @@ describe('quickstart sample tests', () => {
     const stdout = execSync(`node quickstart ${assetName}`);
     assert.include(stdout, assetName);
   });
+
+  it('should search all resources successfully', async () => {
+    const scope = `projects/${process.env.GCLOUD_PROJECT}`;
```

Comment:
If possible, I try to avoid depending on the env var here.  Instead, it's nice to do something like:

```js
const {AssetServiceClient} = require('@google-cloud/asset');
const client = new AssetServiceClient();
const projectId = await client.getProjectId();
```

That way you don't need to set this env var when you run the tests locally, and it can depend on the projectId in the service account credential file.